src/components/BackToTop/BackToTop.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React from 'react';
+import type { ComponentType } from 'react';
+
+import type { Props } from './BackToTop.types';
+import * as Styles from './BackToTop.styles';
+
+import useBackToTopOverlayVisible from 'hooks/useBackToTopOverlayVisible';
+
+const BackToTopLink: ComponentType<Props> = () => {
+  return (
+    <div class="govuk-width-container">
+      <a class="govuk-link govuk-link--no-visited-state" href="#top">
+        <svg role="presentation" focusable="false" class="back-to-top" xmlns="http://www.w3.org/2000/svg" width="13" height="17" viewBox="0 0 13 17">
+          <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
+        </svg>Back to top
+      </a>
+    </div>
+  );
+};
+
+const BackToTop: ComponentType<Props> = ({ mode }: Props) => {
```

Comment:
@MikeCoats Maybe a brief doc for this function as well. Re using it as an overlay or otherwise.